skyuxconfig-schema.json

```diff
@@ -159,6 +159,10 @@
       "description": "Specifies configuration options for communication with SKY UX Host.",
       "type": "object",
       "properties": {
+        "frameOptions": {
+          "description": "Specifies a value for `X-Frame-Options` that will be applied to the SKY UX Host HTTP headers. This value allows SPAs to set their own security preferences for embedded iframes. Possible values include \"sameorigin\", \"deny\", \"blackbaud\", or \"allow-from https://www.example.com\". Setting this value to \"blackbaud\" will only allow iframes to set their `src` attribute to whitelisted Blackbaud domains.",
```

Comment:
@blackbaud-johnly I would appreciate your feedback on my wording!

Reply:
Wondering if we should be this descriptive it the schema?  If we kept in generic, that would also be more inline since the type is `string`.  We could simply give something brief and say for Blackbaud use only, then document the supported values in internal documentation.

Reply:
Based on Bobby's feedback, what do you think of tightening this to:
```Specifies an `X-Frame-Options` value for SKY UX Host HTTP headers to set security preferences for a SPA's embedded iframes. This property is for internal Blackbaud use only.```
We could a corresponding entry in the SKY UX docs at https://developer.blackbaud.com/skyux/learn/reference/configuration that includes a link to more info in the Engineering System docs, and we could create a new entry for this ("Security preferences for embedded iframes") in the SPAs section at https://host.nxt.blackbaud.com/engineering-system-docs/learn/spa.

Reply:
Good feedback.